src/Mvc/Mvc.Cors/src/CorsHttpMethodActionConstraint.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace Microsoft.AspNetCore.Mvc.Cors
 {
-    // Don't casually change the name of this. We reference the full type name in ActionConstraintCache.
```

Comment:
We no longer produce instances of this when doing endpoint routing, so the comment is no longer relevant (I think)

Reply:
Cool 👍 Always good to remove teh uglies